package.json

```
@@ -24,7 +24,9 @@
     "lint:js": "eslint --ext ts,js .",
     "lint:js:fix": "eslint --ext ts,js . --fix",
     "lint:ts": "tsc -p tsconfig.json --noEmit",
-    "prepack": "ember ts:precompile",
+    "prepack": "npm-run-all prepublishOnly:*",
```

Shouldn't this be:

```suggestion
    "prepack": "npm-run-all prepack:*",
```

?